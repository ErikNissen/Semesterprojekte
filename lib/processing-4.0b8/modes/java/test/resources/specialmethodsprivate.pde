version https://git-lfs.github.com/spec/v1
oid sha256:c813f7bfebef930edf989f0a6422be72c617f870c04766936ac13aa02adca53d
size 176
