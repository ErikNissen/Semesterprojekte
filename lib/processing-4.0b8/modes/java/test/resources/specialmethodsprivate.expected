version https://git-lfs.github.com/spec/v1
oid sha256:fce6a4301ba971239353261c8220e0e9764113db403820ff56aef81f1c764c5d
size 933
